sdk/OpenTap.Sdk.MSBuild/OpenTapImageInstaller.cs

```diff
@@ -10,15 +10,33 @@
 
 namespace Keysight.OpenTap.Sdk.MSBuild
 {
+    public interface IImageDeployer
```

Comment:
This is just here to be able to mock things, right? Could we make it internal, and use `InternalsVisibleTo("OpenTap.Package.UnitTests")` ? 

... same for the new public property on `InstallOpenTapPackages`